examples/compute/flavor/min_ram/main.tf

```diff
@@ -0,0 +1,6 @@
+# If the exact amount of RAM is not so important to you, then you can specify the minimum value that will satisfy you 
+# and flavor with minimum of ram will be automatically selected for you.
+data "vkcs_compute_flavor" "basic" {
```

Comment:
include this example into template, and rename resource

Reply:
include and rename to basic_min_ram